aws_pricing/datadog_checks/aws_pricing/data/conf.yaml.example

```diff
@@ -0,0 +1,9 @@
+init_config:
+
+instances:
+    ## @param AmazonEC2 - string - optional
```

Comment:
If the parameter is optional then it should be commented by default, note that you can define a blank instance:

```

instances:
  - 

```